inlong-common/src/main/java/org/apache/inlong/common/util/MaskDataUtils.java

```diff
@@ -0,0 +1,180 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.inlong.common.util;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import org.apache.commons.lang3.StringUtils;
+
+/**
+ * MaskDataUtils is used to mask sensitive information.
+ */
+public class MaskDataUtils {
+
+    private static final List<String> KEYWORDS =
+            Arrays.asList("password", "passphrase", "pwd", "pass", "secret_id", "token",
+                    "secret_key", "secret_token", "secretKey", "publicKey", "secretKey",
+                    "secretId");
```

Comment:
The `secretKey` was duplicated. Suggested changing to:
```
    private static final List<String> KEYWORDS = Arrays.asList(
            "password", "passphrase", "pwd", "pass",
            "token", "secret_token", "secret_id", "secretId",
            "secret_key", "secretKey", "public_key", "publicKey");
```

Reply:
Hello, secretKey and secret_key are different keyword